prog1/implementacoes/uri/1067.py

```diff
@@ -0,0 +1,6 @@
+numero = int(raw_input())
```

Comment:
```suggestion
numero = int(input())
```

Pequena sugestão para mover para `Python 3`. :smile: